packages/react-i18n/CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
   Please see the [migration guide](./migration-guide.md) for more information.
 
+### Fixed
```

Comment:
```suggestion
## [Unreleased]

### Fixed
```

And we'll comment `[Unreleased]` out with the release

Reply:
I think we're just adding a Fixed subsection to an already-existing unreleased section.

Reply:
yeah nvm lol unreleased is already commented out :/